packages/api/src/logger/index.ts

```diff
@@ -243,20 +238,22 @@ export interface RedwoodLoggerOptions {
  */
 export const createLogger = ({
   options,
+  transport,
   destination,
   showConfig = false,
-}: RedwoodLoggerOptions): BaseLogger => {
+}: RedwoodLoggerOptions): pino.BaseLogger => {
   const hasDestination = typeof destination !== 'undefined'
+  const hasTransport = typeof transport !== 'undefined'
```

Comment:
right, we have to decide to keep the idea of a destination or switch to transports.

I think I will need to know the difference between:

```
  transport?: pino.TransportSingleOptions | pino.TransportMultiOptions
   destination?: string | pino.DestinationStream
```

Reply:
If the Pino example is:

https://github.com/pinojs/pino/blob/3937987c03b3dc6eca28806d90fdb3598769f008/examples/transport.js#L9

```
const transport = pino.transport({
  targets: [{
    level: 'warn',
    target: '#pino/file',
    options: {
      destination: file
    }
  }, {
    level: 'info',
    target: 'pino-elasticsearch',
    options: {
      node: 'http://localhost:9200'
    }
  }, {
    level: 'info',
    target: '#pino/pretty'
  }]
})
```

I wonder if setting one destination just creates one transport ... and if set several destinations, several transports.

So, we keep `destination` but convert to transports.

Reply:
I think their idea of a target is RW's idea of a destination.

Reply:
But now each destination/target *also* can accept a level and a set of options.

Reply:
> right, we have to decide to keep the idea of a destination or switch to transports.
> 
> I think I will need to know the difference between:
> 
> ```
>   transport?: pino.TransportSingleOptions | pino.TransportMultiOptions
>    destination?: string | pino.DestinationStream
> ```

Okay, so I think we only need one. The new transport API is to be run in a worker thread, so it does the same thing but multi-thread. Since they already use destinations as an option for transport, should we go with transport only?